docs/items/weapon-configs.md

```diff
@@ -142,4 +142,450 @@ What values a Weapon Type can have:
   * 14+42
 * REQ Tier: 3
 * Game State: 2
-* Notes: -
\ No newline at end of file
+* Notes: -
+  
+## [11] Boxer
+* Weapon Type: CQS48 Bulldog + BR75 Breacher
+* Trait Set: 11
+  * Weapon Damage: 1.0
+* Config: Combo
+* Ammo: Default
+  * 30+90
+* REQ Tier: 4
+* Game State: 3
+* Notes:-
+  
+## [12] Blaze Of Glory
+* Weapon Type: CQS48 Bulldog + Diminisher Of Hope
+* Trait Set: 12
+  * Weapon Damage: 2.90
+* Config: Combo
+* Ammo: Default
+  * 7+7
+* REQ Tier: 5
+* Game State: 4
+* Notes:-
+  
+## [13] Nornfang
+* Weapon Type: S7 Sniper + Diminisher Of Hope
+* Trait Set: 13
+  * Weapon Damage: 1.61
+* Config: Custom
+* Ammo: Boost (14)
+  * 4+6
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+  
+## [14] ppSentinel
+* Weapon Type: Plasma Pistol + Arcane Sentinel Beam
+* Trait Set: 14
+  * Weapon Damage: 1.9
```

Comment:
Make sure the trait scalar values are two decimals accurate, so 1.90 instead of 1.9

Reply:
Okay

Reply:
Has been rectified